core/src/worker/activities/local_activities.rs

```diff
@@ -474,11 +515,54 @@ impl LocalActivityManager {
         }
     }
 
-    pub(crate) async fn shutdown_and_wait_all_finished(&self) {
-        while !self.dat.lock().outstanding_activity_tasks.is_empty() {
+    pub(crate) fn workflows_have_shutdown(&self) {
+        // TODO: Cancel _all_ outstanding las
```

Comment:
Just checking: is the intention to cover this in a distinct PR, or this a forgotten TODO?

Reply:
Actually just not going to do this - LAs should be allowed to finish during shutdown and if lang wants to abort sooner than that, that's up to it.